### DISCUSSION TITLE
Road to 3k Update TODO

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#412] [(C) Suggestions]
- [ ] Fix broken hacks on harmony island
- [x] Hosted arena hacks
- [x] Online Dashboard
- [x] Eval console
- [x] An index page for the extension
- [ ] Easymode
- [ ] Quest hacks
- [x] Toggle click teleporting
- [ ] Triggering battles with certain enemies
- [x] Detailed pet editing
- [x] Morph hacks

**Comment by PatheticMustan** [comment]
Morph hacks: https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/pull/408/commits/3604d6ecf8dc739f7d767f792c807fbb82c029f5

**Comment by PatheticMustan** [comment]
I lied, we're gonna work on revamping WCM first

**Comment by NX721** [comment]
We finished like a very small amount of this.

**Comment by PatheticMustan** [comment]
Damn, I completely forgot about this.

**Reply by NX721** [reply]
Lmao

**Comment by PatheticMustan** [comment]
Gonna move to a new issue
